docs/cn/overview/JobService.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: global
+title: 作业服务器
+group: Overview
+priority: 3
+---
+
+* 内容列表
+{:toc}
+
+## 架构概览
+
+Alluxio 作业服务器是负责将各种不同类型的操作分配给Job Worker的任务调度框架。
+
+Master负责将作业分配为更小的任务，供Job Worker执行并管理作业的完成状态。
+
+Job Worker将来自Job Master的任务排列（queue)，并通过管理可配置的固定线程池(`alluxio.job.worker.threadpool.size`)来完成这些任务。
+
+## 不同类型的作业
+
+### 加载
+
+`fs distributedLoad`CLI命令中使用了加载作业，按特定的副本数将文件加载到Alluxio。
+
+### 迁移
+
+`fs distributedCp`和`fs distributedMv`CLI命令中使用了迁移作业，使用固定的[写入类型]({{ '/en/overview/Architecture.html#data-flow-write' | relativize_url }})进行数据复制/移动。
+
+### 持久化
+
+`fs persist` CLI命令间接使用了持久化作业，以ASYNC_THROUGH[写入类型]({{ '/en/overview/Architecture.html#data-flow-write' | relativize_url }})写入Alluxio时使用持久化作业在后台进行持久化。
```

Comment:
backstick around ASYNC_THROUGH
for all the information that is code instead of human language, using backstick around the code to make it clearer